tdrs-frontend/src/index.js

```diff
@@ -24,8 +24,28 @@ axios.defaults.xsrfCookieName = 'csrftoken'
 axios.defaults.xsrfHeaderName = 'X-CSRFToken'
 axios.defaults.withCredentials = true
 
+function devLogin(devEmail) {
+  const BACKEND_URL = process.env.REACT_APP_BACKEND_URL
+  axios
+    .post(`${BACKEND_URL}/login/cypress`, {
+      username: devEmail,
+      token: 'local-cypress-token',
+    })
+    .then(function (response) {
+      console.log(response)
+    })
+    .catch(function (error) {
+      console.log(error)
+    })
+  store.dispatch({ type: 'SET_AUTH', payload: { devEmail } })
```

Comment:
i think the `payload` passed here needs to be changed to `payload: { email: devEmail }`

what you currently have is javasscript shorthand for `{ devEmail: devEmail }`